src/glossary.rst

```diff
@@ -967,3 +967,117 @@ Glossary
       See also:
 
       * :ref:`Graph Branching`
+
+source workflow
```

Comment:
workflow source?

Reply:
The source workflow is in the workflow source.

Reply:
If you can't play it then surely it's not a workflow?

Reply:
Middle ground: workflow source workflow?

Reply:
Programming analogies might help...

IMO "source code" refers generically to the medium. i.e., that bunch of text is "source code".  It doesn't mean, that is the source for this particular computer program. For that, you would say "program source", not "source program" which sounds like "a program that is the source of something else", not "the source code of the program".

So on that basis, I think we should go with:

*The **workflow source directory" contains the workflow config file that gets installed to (and is therefore the source of) this workflow run directory.*

We don't necessarily have to confuse people by refering to the workflow config file itself as a "workflow source" or "source workflow".